### ISSUE TITLE
Dune builds for coq-elpi and hierarchy-builder

### ISSUE THREAD

**Issue by Alizter** [ejgallego/coq-universe#8]
It would be good to have dune builds of these two coq projects so that we can bring in all of mathcomp.

**Comment by ejgallego** [comment]
Yes, I'm not sure what the status of Dune support for the is, Shachar did some work to have coq-elpi building with Dune https://github.com/jscoq/addon-elpi/tree/2b63902f7721a1abce8bd3f58de7bcb2bb1141b5 but not sure if that will build 100% fine in all cases.

In the long term, we need to support findlib loading, the ball is now on Coq, see https://github.com/coq/coq/issues/16571

**Comment by JasonGross** [comment]
coq/coq#16571 is now closed
